.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '21 1 * * *'
```

Comment:
Just curious, does this mean check any PR that has no activity at 1:21 am? What time zone is it?

Reply:
Yes, good catch. I'm not sure why that is the default time, I didn't set it. The timezone will be UTC for GH action cron schedules, based on a quick Google. In the end, it doesn't matter much when it happens as long as it checks once a day or so.

Reply:
Got it. Thanks!